src/atc/cosmos/cosmos_base_server.py

```diff
@@ -0,0 +1,41 @@
+from typing import Union
+
+from pyspark.sql import DataFrame
+from pyspark.sql.types import DataType
+
+from atc.sql import BaseServer
+
+
+class CosmosBaseServer(BaseServer):
+    def execute_sql(self, sql: str):
+        pass
+
+    def read_table_by_name(self, table_name: str, schema: DataType = None) -> DataFrame:
+        pass
+
+    def read_table(self, table_id: str, schema: DataType = None) -> DataFrame:
+        pass
+
+    def write_table_by_name(
+        self, df_source: DataFrame, table_name: str, rows_per_partition: int = None
+    ):
+        pass
+
+    def write_table(
+        self, df_source: DataFrame, table_id: str, rows_per_partition: int = None
+    ):
+        pass
+
+    def delete_item(
+        self, table_id: str, id: Union[int, str], pk: Union[int, str] = None
+    ):
+        pass
+
+    def delete_container(self, table_id: str):
+        pass
+
+    def delete_container_by_name(self, table_name: str):
+        pass
+
+    def recreate_container_by_name(self, table_name: str):
```

Comment:
Lets be better to make docstrings for methods. For every method there is not in the BaseServer - you could consider making a small docstring.

Reply:
Well.. unless the method is self explaining?